apps/next-evans/lib/friend.ts

```diff
@@ -0,0 +1,23 @@
+import type { gender as Gender } from '@prisma/client';
+import { LANG } from '@/lib/env';
+
+export function getFriendUrl(slug: string, gender: Gender): string {
+  let firstPart = `friend`;
+  if (LANG === `es`) {
+    switch (gender) {
+      case `male`:
+        firstPart = `amigo`;
+        break;
+      case `female`:
+        firstPart = `amiga`;
+        break;
+      case `mixed`:
+        return `/compilations`;
```

Comment:
i don't think this is correct, shouldn't it be `compilaciones`?

also, the english compilations is not handled correctly.  which makes me think, probably for `mixed` gender, probably best to just return `/${friendSlug}` directly, that should always be correct, methinks